src/pagination/PageLink.ts

```diff
@@ -0,0 +1,45 @@
+import { v } from '@dojo/widget-core/d';
+import { DNode } from '@dojo/widget-core/interfaces';
+import { theme, ThemeableMixin, ThemeableProperties } from '@dojo/widget-core/mixins/Themeable';
+import WidgetBase from '@dojo/widget-core/WidgetBase';
+
+import * as css from '../styles/pagination.m.css';
+
+export const PageLinkBase = ThemeableMixin(WidgetBase);
+
+export interface PageLinkProperties extends ThemeableProperties {
+	disabled?: boolean;
+	isArrow?: boolean;
+	label?: string;
+	page: number;
+	onPageRequest?: (page: number) => void;
```

Comment:
In the overall logic, would we really ever have a real world situation where this is optional?